src/sgp/Video.cc

```diff
@@ -97,6 +97,20 @@ static VideoScaleQuality ScaleQuality = VideoScaleQuality::LINEAR;
 static void RecreateBackBuffer();
 static void DeletePrimaryVideoSurfaces(void);
 
+// returns if desktop resolution larger game resolution
+BOOLEAN IsDesktopLargeEnough()
+{
+	SDL_DisplayMode dm;
+	if (SDL_GetDesktopDisplayMode(0, &dm) == 0)
+	{
+		if (dm.w < SCREEN_WIDTH || dm.h < SCREEN_HEIGHT)
+		{
+			return false;
+		}
+	}
```

Comment:
Shouldn't it also return false in the else case?

Reply:
`SDL_GetDesktopDisplayMode` returning non-zero means we do not know about the desktop resolution, so it could be anything. 

I opt for letting it try as it is not (yet) a critical error. It may still run into other problems anyway.

Reply:
`SDL_GetDesktopDisplayMode` returning non-zero means we do not know about the desktop resolution, so it could be anything.

I opt for letting it try as it is not (yet) a critical error. It may still run into other problems anyway.